e2e/pnpm_workspace/lib/b/BUILD.bazel

```diff
@@ -1,9 +1,9 @@
-load("@aspect_rules_js//js:defs.bzl", "js_package")
-load("@npm//:defs.bzl", "link_js_packages")
+load("@aspect_rules_js//npm:defs.bzl", "npm_package")
+load("@npm//:defs.bzl", "link_all_npm_packages")
 
-link_js_packages()
+link_all_npm_packages(name = "node_modules")
 
-js_package(
+npm_package(
```

Comment:
what did we decide about a `package_json` attribute to call out that they ought to be 1:1? do in a followup?

Reply:
we were considering it; under the hood its just syntax sugar but lets chat about it in the public API review